src/main/java/org/commcare/formplayer/services/MediaValidator.kt

```diff
@@ -0,0 +1,58 @@
+package org.commcare.formplayer.services
+
+import com.google.common.collect.ImmutableList
+import java.io.InputStream
+import java.net.URLConnection
+
+/**
+ * Utility functions to validate media attachments in a form
+ */
+object MediaValidator {
+
+    // 3 MB size limit
+    private const val MAX_BYTES_PER_ATTACHMENT = (3 * 1048576 - 1024).toLong()
+    private val SUPPORTED_FILE_EXTS = ImmutableList.of(
+        ".jpg",
```

Comment:
why does just this one have a dot?

Reply:
Maybe typo? I don't think it needs the dot. I've tested it without as part of frontend file type validation and it works fine with or without.